dotCMS/src/main/java/com/dotcms/rest/api/v1/workflow/WorkflowResource.java

```diff
@@ -585,6 +585,8 @@ public final Response saveActionToStep(@Context final HttpServletRequest request
                     ExceptionMapperUtil.createResponse(e, Response.Status.INTERNAL_SERVER_ERROR);
         }
 
+        int[] a = {1};
```

Comment:
Not sure what that line is supposed to do

Reply:
me neither, weird

Reply:
done